cmd/virt-launcher/virt-launcher.go

```diff
@@ -519,12 +519,9 @@ func ForkAndMonitor(containerDiskDir string) (int, error) {
 					log.Log.Reason(err).Errorf("Failed to reap process %d", wpid)
 				}
 
-				// there's a race between cmd.Wait() and syscall.Wait4 when
-				// cleaning up the cmd's pid after it exits. This allows us
-				// to detect the correct exit code regardless of which wait
-				// wins the race.
+				log.Log.Infof("Reaped pid %d with status %d", wpid, int(wstatus))
```

Comment:
This line was added by another commit than the one being cherry-picked here (https://github.com/kubevirt/kubevirt/commit/50c5ede8687f9d749def9466e6acd2ca7cf19066).
It's probably fine for this PR to add it, but only if there's no plan to ever backport that other commit...

Reply:
You're right, better not to mess with it.